server/configmodule.py

```diff
@@ -43,10 +43,12 @@ class DevelopmentConfig(Config):
 
 class MinikubeConfig(Config):
     DEVELOPMENT = True
+    SECRET_PROJECT = 'datcom-website-dev'
     API_ROOT = 'http://127.0.0.1:8081'  # Port for Kubernetes ESP.
 
 
 class GKEConfig(Config):
+    SECRET_PROJECT = 'datcom-website-staging'
```

Comment:
My understanding is that GKE is used by both prod and staging - so should the secret_project be prod (or is this a temp step before we fully migrate prod & staging)? A comment on this would be helpful